src/groovy/org/bbop/apollo/websocket/WebSocketConfig.groovy

```diff
@@ -0,0 +1,33 @@
+package org.bbop.apollo.websocket
+
+import org.bbop.apollo.websocket.HandshakeHandler
+import org.springframework.context.annotation.Configuration
+import org.springframework.messaging.simp.config.MessageBrokerRegistry
+import org.springframework.web.socket.config.annotation.AbstractWebSocketMessageBrokerConfigurer
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry
+
+@Configuration
+@EnableWebSocketMessageBroker
+class WebSocketConfig extends AbstractWebSocketMessageBrokerConfigurer {
+
+    @Override
+    void configureMessageBroker(MessageBrokerRegistry mbr) {
+        mbr.enableSimpleBroker "/queue", "/topic"
+        mbr.setApplicationDestinationPrefixes "/app"
+        mbr.setUserDestinationPrefix "/user/"
+    }
+
+    @Override
+    void registerStompEndpoints(StompEndpointRegistry ser) {
+        ser.addEndpoint("/stomp")
+            .setAllowedOrigins("*")
+            .setHandshakeHandler(handshakeHandler())
+            .withSockJS()
+    }
+
+    public HandshakeHandler handshakeHandler() {
```

Comment:
@garrettjstevens  should this be a factory (createHandleHandler) or a singleton method (getInstance)?